integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -54,15 +54,15 @@ object HiveExample {
       .getOrCreateCarbonSession(
         store, metaStore_Db)
 
-    val carbonHadoopJarPath = s"$rootPath/assembly/target/scala-2.11/carbondata_2.11-1.1" +
```

Comment:
@cenyuhai better to remove this add jar step example wont require any jar to run it,it will create problem with every new release,i corrected this in this pr https://github.com/apache/carbondata/pull/979